back/app/services/side_fx_custom_field_option_service.rb

```diff
@@ -20,6 +20,12 @@ def before_destroy(custom_field_option, _current_user)
   end
 
   def after_destroy(frozen_custom_field_option, current_user)
+    custom_field = frozen_custom_field_option.custom_field
+    if custom_field&.resource_type == 'CustomForm' && custom_field&.input_type == 'select'
```

Comment:
Does this only happen for select type inputs? Not for multiselect or other input types?

Reply:
I think we can only add logic to select input types.